src/Controllers/CopyPatrol.php

```diff
@@ -87,7 +87,37 @@ public function setWikiprojectDao( $wikiprojectDao ) {
 	 * editor_talk_dead: Bool. Is editor talk page non-existent?
 	 */
 	protected function handleGet() {
-		$records = $this->dao->getPlagiarismRecords();
+		$userData = $this->authManager->getUserData();
+		$currentUser = $userData ? $userData->getName() : NULL;
+		$filter = 'open'; // this will be the default
+		$filterUser = NULL;
+
+		// set filter types and descriptions that will be rendered as checkboxes in the view
+		$filterTypes = array(
+			'all' => 'All cases',
+			'open' => 'Open cases',
+			'fixed' => 'All "page fixed" cases',
+			'noaction' => 'All "no action needed" cases'
+		);
+		// add 'My reviews' to filter options if user is logged in
+		if ( isset( $currentUser ) ) {
+			$filterTypes['mine'] = 'My reviews';
+		}
+
+		// use given filter if set, or the default 'open' if filter is 'mine' and user is logged out
+		if ( isset( $_GET['filter'] ) ) {
```

Comment:
We should validate the GET input. Also, check that the input is part of $filterTypes. If it fails the check, show a banner alert (comment below).